### ISSUE TITLE
Feature Request: Support for .msg files (for outlook)

### ISSUE THREAD

**Issue by muratcankurtulus** [microsoft/markitdown#62]
I would like to have support for .msg file for outlook mails. this would be very useful to harvest old experience and data for organizations.

**Comment by afourney** [comment]
Agreed. Also I should create a meta-issue for tracking requested formats.

**Comment by makermotion** [comment]
A question for contributions, since OLE is kind of hard to parse without using 3rd party packages. Is it ok to use 3rd parties for this purpose. Because some may not have fully permissible licenses like GPL?

**Comment by afourney** [comment]
> A question for contributions, since OLE is kind of hard to parse without using 3rd party packages. Is it ok to use 3rd parties for this purpose. Because some may not have fully fully permissive licenses like GPL?

Good question. I *think* LGPL should be fine. GPL gets more problematic. Propose a new dependency, and we can take a look. We can also open things up to allow you to register your own converters from outside the core library -- I think we'll do this anyways. In which case, you are free to create and distribute a converter extension independently with whatever license is wanted or necessary.

**Comment by makermotion** [comment]
I was considering to propose **_read_/_parse_** `.msg` files using [msg-extractor](https://github.com/TeamMsgExtractor/msg-extractor) but like I said it's GPLv3.

**Comment by makermotion** [comment]
created a pull request for this. [#196 ](https://github.com/microsoft/markitdown/pull/196)

**Comment by sakshamgoyal-01** [comment]
> I was considering to propose **_read_/_parse_** `.msg` files using [msg-extractor](https://github.com/TeamMsgExtractor/msg-extractor) but like I said it's GPLv3.

https://github.com/TeamMsgExtractor/msg-extractor/issues/431#issuecomment-2460726148
> As I understand it, the GPL license would allow you to use it through the command line interface using subprocess.run(). However you should not import extract_msg in your codebase.

I would strongly considering using this to add additional functionality to the extractor like for #1567
